src/yaml.ts

```diff
@@ -45,13 +45,14 @@ export const setExtraDeps = async (
   doc: Document,
   extraDeps: Package[]
 ): Promise<Document> => {
-  const seq = doc.get('extra-deps') as YAMLSeq
-  // eslint-disable-next-line github/array-foreach
-  extraDeps.forEach((dep, index) => {
-    seq.set(index, `${dep.name}-${dep.version}`)
-  })
-  doc.set('extra-deps', seq)
-
+  if (extraDeps.length && doc.has('extra-deps')) {
```

Comment:
If I understand you correctly, we only need to check `extraDeps.length`, if true the other condition should be true as well? and vice versa.